pkg/apis/kops/validation/validation.go

```diff
@@ -1046,6 +1048,9 @@ func validateRollingUpdate(rollingUpdate *kops.RollingUpdate, fldpath *field.Pat
 		} else if surge < 0 {
 			allErrs = append(allErrs, field.Invalid(fldpath.Child("maxSurge"), rollingUpdate.MaxSurge, "Cannot be negative"))
 		}
+		if unavailable == 0 && surge == 0 {
+			allErrs = append(allErrs, field.Forbidden(fldpath.Child("maxSurge"), "Cannot be zero if maxUnavailable is zero"))
```

Comment:
This is what we told users to do before, IIUC.  Should we have a hint like "set drainAndTerminate: false to disable rolling updates" ?

But ... I don't think we can do this, if the object was previously valid I think our apimachinery requires that it still be valid.  We should see what happens, but I suspect a warning is going to be easier for everyone here (at least for this api version, in the next api version we can map 0/0 to drainAndTerminate=false, I think)